src/test/compile-fail/deriving-meta-unknown-trait.rs

```diff
@@ -11,7 +11,7 @@
 // ignore-tidy-linelength
 
 #[derive(Eqr)]
-//~^ ERROR cannot find derive macro `Eqr` in this scope
+//~^ ERROR cannot find macro `Eqr` in this scope
```

Comment:
Not sure that this is an improvement -- I think we should use "derive macro" (status quo) or "custom derive" here.
cc @nrc